Add instructions and script on how to use the lumbar rootlets model

valosekj opened the pull request:
This PR adds instructions and script on how to use the lumbar rootlets model (alpha version).

jcohenadad commented on packaging_lumbar_rootlets/README.md:
```diff
@@ -0,0 +1,69 @@
+⚠️ This README provides instructions on how to use the model for **_lumbar_** rootlets. 
+Please note that this model is still under development and is not yet available in the Spinal Cord Toolbox (SCT).
+
+⚠️ For the stable model for dorsal cervical rootlets only, use SCT v6.2 or higher (please refer to this [README](..%2FREADME.md)).
```
```suggestion
> [!NOTE]
> If you would like to use the model for _dorsal_ cervical rootlets only, use SCT v6.2 or higher (please refer to this [README](..%2FREADME.md)).
```

jcohenadad commented on packaging_lumbar_rootlets/README.md:
```diff
@@ -0,0 +1,69 @@
+
+- [Spinal Cord Toolbox (SCT)](https://spinalcordtoolbox.com/user_section/installation.html)
+- [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) 
+- Python
```
version requirement needed

jcohenadad commented on packaging_lumbar_rootlets/README.md:
```diff
@@ -0,0 +1,69 @@
+ 
+### Step 3: Getting the Predictions
+
+ℹ️ To temporarily suppress warnings raised by the nnUNet, you can run the following three commands in the same terminal session as the above command:
```
```suggestion
> [!NOTE]
> To temporarily suppress warnings raised by the nnUNet, you can run the following three commands in the same terminal session as the above command:
> ```bash
> export nnUNet_raw="${HOME}/nnUNet_raw"
> export nnUNet_preprocessed="${HOME}/nnUNet_preprocessed"
> export nnUNet_results="${HOME}/nnUNet_results"
> ```
```
I'm not sure this formatting is going to work...

valosekj replied:
Yes, it is working! https://github.com/ivadomed/model-spinal-rootlets/blob/main/packaging_lumbar_rootlets/README.md#step-3-getting-the-predictions

jcohenadad commented on packaging_lumbar_rootlets/README.md:
````diff
@@ -0,0 +1,69 @@
+python packaging_lumbar_rootlets/run_inference_single_subject.py -i sub-001_T2w.nii.gz -o sub-001_T2w_label-rootlets_dseg.nii.gz -path-model ~/Downloads/Dataset202_LumbarRootlets_r20240527 -fold 0
+```
+
+ℹ️ The script also supports getting segmentations on a GPU. To do so, simply add the flag `--use-gpu` at the end of the above commands. 
+By default, the inference is run on the CPU. It is useful to note that obtaining the predictions from the GPU is significantly faster than the CPU.
````
```suggestion
> [!NOTE]
> The script also supports getting segmentations on a GPU. To do so, simply add the flag `--use-gpu` at the end of the above commands. By default, the inference is run on the CPU. It is useful to note that obtaining the predictions from the GPU is significantly faster than the CPU.
```

valosekj commented:
@jcohenadad, thanks for the suggestions. I apologize for merging the PR too quickly. So, I have addressed your comments in [this commit](https://github.com/ivadomed/model-spinal-rootlets/commit/7b3cd8b9275e1a18da975a7cbbabea08552de9b8).
